kcidb/db/sqlite/__init__.py

```diff
@@ -122,8 +123,8 @@ def init(self):
                         raise Exception(
                             f"Failed creating table {name!r}"
                         ) from exc
-                version = int(io.schema.LATEST.major * 1000 +
-                              io.schema.LATEST.minor % 1000)
+                version = int(IO_LATEST_VERSION.major * 1000 +
+                              IO_LATEST_VERSION.minor % 1000)
```

Comment:
Same here regarding calling it `IO_LATEST_VERSION`. Besides, it's not the version of "I/O", it's the version of "I/O schema".